addon-test-support/index.js

```diff
@@ -0,0 +1,30 @@
+import require from 'require';
+import { hasRunAllDisposables } from 'ember-lifeline';
+
+let QUnit = require('qunit').default;
```

Comment:
https://github.com/rwjblue/ember-lifeline/pull/68#discussion_r172581655 is still an issue here I believe